src/NServiceBus.Persistence.DynamoDB/Installer.cs

```diff
@@ -6,66 +6,19 @@
     using System.Threading.Tasks;
     using Amazon.DynamoDBv2;
     using Amazon.DynamoDBv2.Model;
-    using Logging;
 
     class Installer
     {
-        public Installer(IProvideDynamoDBClient clientProvider, InstallerSettings settings,
-            OutboxPersistenceConfiguration outboxConfiguration)
+        public Installer(IAmazonDynamoDB client)
         {
-            installerSettings = settings;
-            this.outboxConfiguration = outboxConfiguration;
-            this.clientProvider = clientProvider;
+            this.client = client;
         }
 
-        public async Task Install(CancellationToken cancellationToken = default)
-        {
-            if (installerSettings == null || installerSettings.Disabled)
-            {
-                return;
-            }
-
-            if (installerSettings.CreateOutboxTable)
-            {
-                try
-                {
-                    await CreateOutboxTableIfNotExists(clientProvider.Client, outboxConfiguration.TableName,
-                        cancellationToken).ConfigureAwait(false);
-                }
-                catch (Exception e) when (!(e is OperationCanceledException &&
-                                            cancellationToken.IsCancellationRequested))
-                {
-                    log.Error(
```

Comment:
do we want to keep this log entry? If yes we should add it to https://github.com/Particular/NServiceBus.Persistence.DynamoDB/pull/187/files#diff-d00d898f06b3d7057360c176613bc598da6f39019816d8508c2948aed0a8b23dR29 instead